cloudbuild.yaml

```diff
@@ -93,6 +93,7 @@ steps:
         docker push $$DOCKER_REPO:$COMMIT_SHA
         docker tag $$SNUBA_IMAGE $$DOCKER_REPO:nightly
         docker push $$DOCKER_REPO:nightly
+images: ['us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA']
```

Comment:
Can you provide more details on why this is needed for Freight?

Reply:
Added comment and updated description.